### PR TITLE
Add Erlang interoperability to Portuguese (Brazil)

### PR DESCRIPTION
**Pull request by philss**
Related to issue #122.

### CODE REVIEW

File: pt/lessons/advanced/erlang.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+---
+layout: page
+title: Interoperabilidade com Erlang
+category: advanced
+order: 1
+lang: pt
+---
+
+Um dos benefícios adicionais em se construir em cima da ErlangVM é a abundância de bibliotecas existentes disponíveis para nós. A interoperabilidade nos permite usar essas bibliotecas e a biblioteca padrão Erlang a partir do nosso código Elixir. Nessa lição, nós vamos olhar como acessar funcionalidades da biblioteca padrão juntamente com pacotes Erlang de terceiros.
```
Comment (pragmaticivan) [inline pt/lessons/advanced/erlang.md]: I think instead `vamos olhar` you can use `vamos ver`.

File: pt/lessons/advanced/erlang.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+- [Biblioteca padrão](#biblioteca-padrao)
+- [Pacotes Erlang](#pacotes-erlang)
+- [Diferenças notáveis](#diferencas-notaveis)
+  - [Atoms](#atoms)
+  - [Strings](#strings)
+  - [Variáveis](#variaveis)
+
+
+## Biblioteca padrão
```
Comment (pragmaticivan) [inline pt/lessons/advanced/erlang.md]: You should use `## <a name="biblioteca-padrao"></a>Biblioteca padrão` when the title has accentuation.

File: pt/lessons/advanced/erlang.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+Agora podemos acessar nossa biblioteca Erlang:
+
+```elixir
+png = :png.create(#{:size => {30, 30},
+                    :mode => {:indexed, 8},
+                    :file => file,
+                    :palette => palette}),
+```
+
+## Diferenças notáveis
```
Comment (pragmaticivan) [inline pt/lessons/advanced/erlang.md]: You should use link name here as well.

File: pt/lessons/advanced/erlang.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+Agora que sabemos como usar Erlang nós devemos cobrir alguns contrapontos que vem com a interoperabilidade com Erlang.
+
+### Atoms
```
Comment (pragmaticivan) [inline pt/lessons/advanced/erlang.md]: Maybe convert the title to `Átomos`?

File: pt/lessons/advanced/erlang.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+** (FunctionClauseError) no function clause matching in :string.strip_left/2
+    (stdlib) string.erl:380: :string.strip_left("Hello World", 32)
+    (stdlib) string.erl:378: :string.strip/3
+    (stdlib) string.erl:316: :string.words/2
+
+iex> "Hello World" |> to_char_list |> :string.words
+2
+```
+
+### Variáveis
```
Comment (pragmaticivan) [inline pt/lessons/advanced/erlang.md]: Here too. a with `name` attribute.

**Comment by pragmaticivan** [comment]
@philss just left some comments, thanks! o/

File: pt/lessons/advanced/erlang.md (hunk 1 above)
Comment (philss) [inline pt/lessons/advanced/erlang.md]: 5955b32

File: pt/lessons/advanced/erlang.md (hunk 2 above)
Comment (philss) [inline pt/lessons/advanced/erlang.md]: 5955b32

File: pt/lessons/advanced/erlang.md (hunk 3 above)
Comment (philss) [inline pt/lessons/advanced/erlang.md]: 5955b32

File: pt/lessons/advanced/erlang.md (hunk 4 above)
Comment (philss) [inline pt/lessons/advanced/erlang.md]: 5955b32

File: pt/lessons/advanced/erlang.md (hunk 5 above)
Comment (philss) [inline pt/lessons/advanced/erlang.md]: 5955b32

**Comment by philss** [comment]
@pragmaticivan thanks, man! All answered. Please check again.

**Comment by pragmaticivan** [comment]
:sparkles: Nice! Thanks @philss

**Pull request merged**